Assets/Scripts/NursePatrol.cs

```diff
@@ -6,6 +6,9 @@
 public class NursePatrol : MonoBehaviour
 {
     public Transform[] points;
+    public GameObject spy;
```

Comment:
Maybe we can make this more general and just name it `target`?
Also since we're only using the transform of this GameObject we can make this Transform type instead right?

Reply:
Yes! I was thinking about that, we should have a general name for all enemies/spy variables throughout our files for consistency. Okay yup I can change that into a Transform Type!